code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -218,10 +218,16 @@
 	force_threshold = 10
 	pressure_resistance = 50
 	mob_size = MOB_SIZE_LARGE
-	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	var/independent = FALSE
 
+/mob/living/simple_animal/hostile/blob/blobbernaut/Login()
+	. = ..()
+	if (hud_used)
```

Comment:
then use ```/mob/Login()```? my point is just contain this code so it only exists in one place, instead of a lot of ```Login()``` boilerplate